README.md

```diff
@@ -1,38 +1,38 @@
-The C1T2X radios function similar to the DSRC radios used by CARMA Platform equipped vehicles in that they receive UDP packets from a PC and broadcast them "over the air." The radios used by the CARMA Platform are configured to be in Road Side Unit (RSU) mode where messages are forwarded, and C1T2X seeks to emulate this behavior.
+The V2X radios function similar to the DSRC radios used by CARMA Platform equipped vehicles in that they receive UDP packets from a PC and broadcast them "over the air." The radios used by the CARMA Platform are configured to be in Road Side Unit (RSU) mode where messages are forwarded, and V2X seeks to emulate this behavior.
 
-The C1T2X radios are intended to use the same driver as the full-scale CARMA vehicle's DSRC radios, specifically the [carma-cohda-dsrc-driver](https://github.com/usdot-fhwa-stol/carma-cohda-dsrc-driver).
+The V2X radios are intended to use the same driver as the full-scale CARMA vehicle's DSRC radios, specifically the [carma-cohda-dsrc-driver](https://github.com/usdot-fhwa-stol/carma-cohda-dsrc-driver).
 
-The C1T2X solution uses the WiFi band for its Vehicle Area Network (VANET) rather than Dedicated Short Range Communications (DSRC) or Cellular V2X (C-V2X). It is intended to be an educational tool used to facilitate communication and cooperation between scaled-down vehicles and infrastructure - and it not intended as a deployable solution. Public Deployment of a WiFi-based VANET is outside of the scope of the C1T project, and may be susceptible to restrictions/guidelines from the Federal Communications Commission (FCC).
+The V2X solution uses the WiFi band for its Vehicle Area Network (VANET) rather than Dedicated Short Range Communications (DSRC) or Cellular V2X (C-V2X). It is intended to be an educational tool used to facilitate communication and cooperation between scaled-down vehicles and infrastructure - and it not intended as a deployable solution. Public Deployment of a WiFi-based VANET is outside of the scope of the CDA1tenth project, and may be susceptible to restrictions/guidelines from the Federal Communications Commission (FCC).
 
-C1T2X radios are capable of running their own applications through threading - provided that message decoding/encoding and parsing is enabled.
+V2X radios are capable of running their own applications through threading - provided that message decoding/encoding and parsing is enabled.
 At this time, the radios do not decode or encode packets and function only to forward messages - similar to the full scale CARMA Platform vehicles' radios. However, further work can be done to enable this.
 
-At a high level, the C1T2X radios can:
+At a high level, the V2X radios can:
 - Receive UDP packets over the LAN from the Jetson Xavier
 - Broadcast UDP packets over the VANET to other scaled-down cooperative entities
 - Receive UDP packets over the VANET from other scaled-down cooperative entities
 - Broadcast UDP packets over the LAN to the Jetson Xavier
```

Comment:
Update 1) and 4) to be be "connected vehicle computer" or similar. Or just Jetson Orin Nano but may be needlessly specific